### PR TITLE
Adding -Interactive param to manually set PortalAliases during RestoreDnnSite

### LINKED ISSUE #19: Restore-DNNSite should take a map of aliases to rename
When restoring a site with multiple domains, it would be nice to be able to specify custom domains.

Or, alternatively, optionally prompt for names for the aliases.

### PR DESCRIPTION
**Pull request by engage-chancock**
Resolves #19 (through the alternative approach)

This adds a parameter `-Interactive` which, when used, will prompt the user to manually rename each PortalAlias.

If a user expects that the passed `-Domain` parameter won't match every PortalAlias in the database (i.e. if a site has foo.org and bar.com) they can pass this parameter to rename each alias during the RestoreDNNSite procedure.

### CODE REVIEW

File: DnnWebsiteManagement/DnnWebsiteManagement.psm1 [hunk 1]
```diff
@@ -599,8 +604,30 @@ function New-DNNSite {
...
+        $ProcessManual = ''
+
+        if ($Interactive.IsPresent) {
+          $ProcessManual = Read-Host -Prompt "Would you like to manually rename all $aliasCount portal aliases? (y/n)"
+          while("y","n" -notcontains $ProcessManual ) {
+            $ProcessManual = Read-Host -Prompt "Would you like to manually rename all $aliasCount portal aliases? (y/n)"
+          }
+        }
+        
+        if ($ProcessManual -eq 'y') {
```
Comment (bdukes) [inline DnnWebsiteManagement/DnnWebsiteManagement.psm1:618]: it looks to me like if I answer `'n'`, the automatic renaming is skipped

**Review by bdukes — APPROVED** [review]
thanks!

File: DnnWebsiteManagement/DnnWebsiteManagement.psm1 (hunk 1 above)
Comment (engage-chancock) [reply DnnWebsiteManagement/DnnWebsiteManagement.psm1:618]: The automatic renaming is handled in the else on line 628

Comment (bdukes) [reply DnnWebsiteManagement/DnnWebsiteManagement.psm1:618]: ah, I was mismatching the braces, thanks for setting me straight

**Comment by bdukes** [comment]
[Version 1.8.0](https://www.powershellgallery.com/packages/DnnWebsiteManagement/1.8.0) is released with your changes. Thanks @engage-chancock!

**Pull request merged**